core/commands/show_commit.py

```diff
@@ -8,18 +8,23 @@
 
 class GsShowCommitCommand(WindowCommand, GitCommand):
 
-    def run(self, commit_hash):
+    def run(self, commit_hash, show_file=False, filepath=None):
```

Comment:
You don't need both `show_file` and `filepath` here.  Reduce this to a single variable.